modules/web-console/frontend/app/core/admin/Admin.data.ts

```diff
@@ -111,4 +111,18 @@ export default class IgniteAdminData {
             .then(({ data }) => data)
             .catch(({data}) => {throw data;});
     }
+
+    resetFailedAttempts(user: User) {
+        return this.$http
+            .patch(`/api/v1/admin/users/${user.id}`, {resetFailedSignInAttempts: true})
+            .then(() => {
+                user.failedLoginAttempts = 0;
+                user.lastFailedLogin = 0;
```

Comment:
Am I right that this resets values on frontend? What if server responded with 200 but for some reason failed to update the numbers? Shouldn't them update via websockets?

Reply:
server can't respond with 200 and doesn't update fields